tox.ini

```diff
@@ -48,34 +66,11 @@ deps =
     botocore==1.3.23
     Pillow==3.4.2
 
-[testenv:py36]
-basepython = python3.6
-
-[testenv:py37]
-basepython = python3.7
-
-[testenv:py38]
-basepython = python3.8
```

Comment:
Is there a way now to run tests for multiple interpreters locally?

Reply:
Yes, `py35`, `py36` and so on still work as expected nonetheless (notice I left them being used in Travis CI). `tox` seem to handle them “magically”.